vispy/visuals/xyz_axis.py

```diff
@@ -10,17 +10,23 @@ class XYZAxisVisual(LineVisual):
     x=red, y=green, z=blue.
     """
     def __init__(self, **kwargs):
-        verts = np.array([[0, 0, 0],
-                          [1, 0, 0],
-                          [0, 0, 0],
-                          [0, 1, 0],
-                          [0, 0, 0],
-                          [0, 0, 1]])
-        color = np.array([[1, 0, 0, 1],
-                          [1, 0, 0, 1],
-                          [0, 1, 0, 1],
-                          [0, 1, 0, 1],
-                          [0, 0, 1, 1],
-                          [0, 0, 1, 1]])
-        LineVisual.__init__(self, pos=verts, color=color, connect='segments',
-                            method='gl', **kwargs)
+        if 'pos' not in kwargs:
```

Comment:
@GuillaumeFavelier Not sure which version is more pythonic, but as `kwargs` is a dict, we could use something like:

```kwargs['pos'] = kwargs.pop('pos', default)```

Reply:
I don't know either which is more pythonic but I can definitely use this syntax. And in this case, all the defaults need to be defined beforehand I suppose?

Reply:
@GuillaumeFavelier After thinking a bit, there would be this solution:

```python
pos = np.array([[0, 0, 0],
                [1, 0, 0],
                [0, 0, 0],
                [0, 1, 0],
                [0, 0, 0],
                [0, 0, 1]])
kwargs.update(pos=kwargs.pop('pos', pos)
```
And same with the other kwargs. @larsoner Is there some vispy-style how to do this kind of kwarg-handling?

Reply:
I vaguely recall seeing `.pop`'s used, @kmuehlbauer your code looks good

Reply:
What's wrong with `kwargs.setdefault('pos', pos)`?

Reply:
@djhoese Believe me or not, I did not know, that something like this exist. This is by far the most minimal solution.